### PR TITLE
Sample yaml files for running retagger on Kubernetes

### PR DESCRIPTION
**Pull request by priyawadhwa**

### CODE REVIEW

File: reconciletags/examples/README.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+`kubectl create secret generic retagger-secret  --from-file=[PATH TO retagger_secret.json]`
+
+To ensure the secret was created, run
+
+`kubectl get secrets`
+
+retagger-secret should be in the output list of secrets.
+
+__Activate Service Account in gCloud__
```
Comment (dlorenc) [inline reconciletags/examples/README.md]: I think they prefer to refer to it as "the Cloud SDK"

Comment (dlorenc) [reply reconciletags/examples/README.md]: Actually though, I don't think you need this step. It was just for permissions and debugging.

File: reconciletags/examples/retagger_cronjob.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            - name: GOOGLE_APPLICATION_CREDENTIALS
+              value: /secret/retagger_secret.json
+          volumes:
+          - name: git-volume
+            gitRepo:
+              repository: [LINK TO YOUR GIT REPOSITORY]
+          - name: retagger-secret
+            secret:
+              secretName: retagger-secret
+          restartPolicy: OnFailure
```
Comment (dlorenc) [inline reconciletags/examples/retagger_cronjob.yaml]: nit newlines

**Comment by aaron-prindle** [comment]
friendly ping, wondering if this PR is still being considered

**Pull request closed without merging**